bot/cogs/fun.py

```diff
@@ -94,6 +95,30 @@ def __init__(self, bot: Bot):
         if "revive" in message.content.lower():
             await emojify(message, "nou")
 
+        # Ask if user has contacted support before letting them ping staff
+        if self.staff_role is None:
+            for y in self.bot.get_server(SERVER_ID).roles:
+                if y.id == STAFF_ROLE_ID:
+                    self.staff_role = y
+                break
```

Comment:
Shouldn't this be indented?

Reply:
@ddevlop Shouldn't this `break` statement be part of the `if` block?